src/shared/GameRound/GameRoundAdmin.tsx

```diff
@@ -101,6 +102,9 @@ const GameRoundAdmin = (props: any) => {
   };
 
   const setAllWrong = () => {
+    if (counterWrongAnswers != 3){
+      playSound(wrongAnswerSound, 5000);
```

Comment:
I suggest adding a constant of
WRONG_ANSWER_TIME = 3000 in constants.ts

Try and apply it both on the client where the X is shown and also here on the dashboard where the sound is played